install_requirements.py

```diff
@@ -78,6 +78,14 @@ def python_is_compatible():
         else:
             print(f"Error: {arg} must follow --pybind")
             sys.exit(1)
+    elif arg == "--clean":
+        print(f"Cleaning build artifacts...")
+        if sys.platform == "win32":
+            subprocess.run(["rmdir", "/s", "cmake-out*/"], check=True)
+            subprocess.run(["rmdir", "/s", "pip-out/"], check=True)
+        else:
+            subprocess.run(["rm", "-rf", "cmake-out*/"], check=True)
+            subprocess.run(["rm", "-rf", "pip-out/"], check=True)
```

Comment:
I suggest exiting at this point. "clean" commands typically clean-then-exit, not clean-then-build.

And maybe complain and exit non-zero if there are any commandline args other than "--clean", so that scripts don't say "install_requirements.sh --clean --pybind" and think that it successfully built.